examples/express-graphql-http-functions/.gitignore

```diff
@@ -0,0 +1 @@
+dist/*.js
```

Comment:
Any idea why we need this here and not for the other examples?

Reply:
Not really necessary, nothing breaks with our without this. It's just nice (IMO) to hide/ignore the build output... I could remove it, or add this to the other examples, what do you think?

Reply:
Hmm, for some reason the dist files in the other projects get ignored for me locally. Maybe this entry is ignoring all of them for me? https://github.com/captbaritone/grats/blob/6753bc27e316dc87ff36e456809a65e46833c903/.gitignore#L2

I wonder why it's different for you.